bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/security.xml

```diff
@@ -26,7 +26,7 @@
 	<channel-group-type id="siren">
 		<label>Siren Status</label>
 		<channels>
-			<channel id="status" typeId="siren-status"/>
+			<channel id="status" typeId="siren-status-ro"/>
```

Comment:
Maybe you should avoid renaming the channel type, I am not sure what could be the impact on existing channels (when thing was created through UI).
@jlaur is more expert than me on that subject I believe. WDYT ?

Reply:
I believe you need to create update instructions with `update-channel` according to https://www.openhab.org/schemas/update-description-1.0.0.xsd. Perhaps @J-N-K can confirm.

Reply:
Ok, I reverted to siren-status the read-only version, only adding siren-status-rw

Reply:
This will be another beast because I do not use xml defined things in this binding.